watson/cli.py

```diff
@@ -1341,11 +1341,8 @@ def config(context, key, value, edit):
     wconfig = watson.config
 
     if edit:
-        try:
-            with open(watson.config_file) as fp:
-                rawconfig = fp.read()
-        except (IOError, OSError):
-            rawconfig = ''
+        with open(watson.config_file) as fp:
+            rawconfig = fp.read()
```

Comment:
I think that's still a nice thing to be resilient in case we can't find the config file.

Reply:
Nice suggestion.